### ISSUE TITLE
Clicking Author Name in the Title Bar Should Display the Author Bio

### ISSUE THREAD

**Issue by owlboy** [oel-mediateam/sbplus_v3#8]
If clicking the author name in the title bar could jump right to the author profile that would be great.

**Comment by Lin87** [comment]
I thought for accessibility reason we don't want it to do that. My memory could be wrong, but I thought we had a discussion on the title/black bar issues during the accessibility testing.

**Comment by owlboy** [comment]
Ok, that sounds good.
